src/interfaces/index.ts

```diff
@@ -86,3 +90,136 @@ export type DocumentContextWithSession = DocumentContext & {
 };
 
 export type IncomingMessageWithSession = IncomingMessage & Session;
+
+export interface SessionContents {
+    body: any;
+}
```

Comment:
We should remove this `SessionContents` as its no longer used

Reply:
Now removed.